libqtile/layout/columns.py

```diff
@@ -103,23 +111,30 @@ class Columns(Layout):
         ("name", "columns", "Name of this layout."),
         ("border_focus", "#881111", "Border colour for the focused window."),
         ("border_normal", "#220000", "Border colour for un-focused windows."),
+        ("border_focus_stack", "#881111",
+         "Border colour for the focused window in stacked columns."),
+        ("border_normal_stack", "#220000",
+         "Border colour for un-focused windows in stacked columns."),
         ("border_width", 2, "Border width."),
         ("margin", 0, "Margin of the layout."),
-        ("autosplit", True, "Autosplit newly created columns."),
+        ("split", True, "New columns presentation mode."),
```

Comment:
Is this potential config breakage worth it? What's the rationale?

Reply:
This is just to get in line with the Wmii layout naming scheme, but not strictly necessary.
(Also `split` is shorter and describes better the setting than `autosplit`.)